ansible/roles/marathon-server/templates/marathon.conf.j2

```diff
@@ -27,6 +27,6 @@ exec /usr/bin/marathon {% if (ssl_keystore_pass is defined) and not(ssl_keystore
 
 post-start script
   sleep 60
-  curl -X POST -H "Content-Type: application/x-www-form-urlencoded" -u {{ marathon_admin_user }}:{{ marathon_admin_password }} --data "logger=mesosphere.marathon.example&level=OFF" http://{{ marathon_webui_host }}/logging
+  curl -X POST -H "Content-Type: application/x-www-form-urlencoded" -u {{ marathon_admin_user }}:{{ marathon_admin_password }} --data "logger=mesosphere.marathon.example&level=OFF" {{ bamboo_marathon_proxy_scheme|default('http') }}://{{ marathon_webui_host }}/logging
```

Comment:
The request by any non-leader gets forwarded to the leader, and changes to the leaders reflects for all the masters.